PIE-7788: Update pipeline-chaining.md (#685)
harness/developer-hub: docs/platform/8_Pipelines/pipeline-chaining.md

   For more information on access, see [Role-based access control](../4_Role-Based-Access-Control/1-rbac-in-harness.md)

   ![](./static/pipeline-chain-list.png)
   
   :::note

>    For a succesfull execution, user must have execute access to both parent & child pipeline.

Reviewer: ```suggestion
   You must have execute permissions for the parent and child pipeline to ensure successful execution.
```
